static/src/javascripts/boot.js

```diff
@@ -151,28 +151,34 @@ const go = () => {
             }
         });
 
-        const fakeBootCommercial = { bootCommercial: () => { } };
-        const commercialBundle = () =>
-            config.get('switches.standaloneCommercialBundle') && !config.get('page.isHosted', false)
-                ? loadScript(config.get('page.commercialBundleUrl')).then(
-                    () => fakeBootCommercial,
-                )
-                : import(
-                    /* webpackChunkName: "commercial" */
-                    'bootstraps/commercial-legacy'
-                );
+        // Start downloading these ASAP
 
+		const fetchCommercial = () => {
```

Comment:
Why has this become a method/function rather than a constant?

Reply:
in part to avoid having to do things like `(markTime('commercial request'), commercialBundle())`, and also having to disable `no-nested-ternary`. 

allows you to not trigger anything until you know you'll need it while trying to make it as legible as possible